packages/e2e-tests/config/setup-debug-artifacts.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const util = require( 'util' );
+const root = process.env.GITHUB_WORKSPACE || __dirname + '/../../../';
+const ARTIFACTS_PATH = root + '/artifacts';
```

Comment:
Should we add `artifacts` to `.gitignore` for local failures?

Reply:
Good note! Addressed in https://github.com/WordPress/gutenberg/pull/26957